torch/_dynamo/debug_utils.py

```diff
@@ -243,7 +243,8 @@ class AccuracyError(Exception):
             textwrap.dedent(
                 f"""
                 compiled = {COMPILER_REPRO_OPTIONS[compiler_name][1]}(mod, args)
-                compiled(args)
+                ref = compiled(args)
+                torch.cuda.synchronize() # Ensures that segfaults are surfaced
```

Comment:
you should only call `torch.cuda.synchronize()` if `torch.cuda.is_available()`

Reply:
for CPU minifications, this wont apply

Reply:
Good catch! Will send a change.